spec/class_book_spec.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../class_books_handler'
+require_relative '../class_genres_handler'
+require_relative '../class_labels_handler'
+require_relative '../class_authors_handler'
+
+RSpec.describe BookHandler do
+  let(:book_handler) { BookHandler.new }
```

Comment:
- [ ] Please make sure to add a test for the method `can_be_archived?()` in the Book class. :pray:

Reply:
thank you. @MoyasiGinko is goint to fixit